pages/index.js

```diff
@@ -44,6 +54,21 @@ export default function Home() {
           </button>
         ))}
       </div>
+      <div style={{ marginTop: "12px" }}>
+        <label for="manual-wallet">
+          Manually set "discovery.wallet" config:{" "}
+        </label>
+        <input ref={discoveryWalletInputRef} name="manual-wallet"></input>
+        <button
+          onClick={async () =>
+            await fcl
+              .config()
+              .put("discovery.wallet", discoveryWalletInputRef?.current?.value)
+          }
+        >
+          Set
+        </button>
+      </div>
       <pre>{JSON.stringify({ currentUser, config }, null, 2)}</pre>
```

Comment:
This seems to be blowing up after authn with dev-wallet

```
Error: Objects are not valid as a React child
React child (found: object with keys {f_type, f_vsn, addr, cid, loggedIn, expiresAt, services})

The above error occurred in the <pre> component:
```

Reply:
Fixed!